spec/support/matchers/general_matchers.rb

```diff
@@ -26,3 +26,24 @@
     msg
   end
 end
+
+RSpec::Matchers.define :be_json do
+  match do |actual|
+    JSON.parse(actual)
```

Comment:
**[nit pick/not a blocker]** if the `actual` variable raises some exception different from JSON parsing (e.g calls a method on nil, claim.some_method) this matcher will tell the JSON is not parsable which is not the case. The rescue should probably not rescue from StandardError but from JSON::ParserError.